Src/library/net/Clib/network.c

```diff
@@ -775,11 +778,108 @@ EIF_INTEGER c_put_stream_noexception(EIF_INTEGER fd, EIF_POINTER stream_pointer,
 }
 
 void c_put_stream(EIF_INTEGER fd, EIF_POINTER stream_pointer, EIF_INTEGER length)
-	/*x transmission of string s of size size trought socket fd */
+	/*x transmission of string s of size length throught socket fd */
 {
 	eif_net_check(c_put_stream_noexception(fd, stream_pointer, length));
 }
 
+EIF_INTEGER c_unoptimized_sendfile_noexception(EIF_INTEGER out_fd, FILE* f, EIF_INTEGER length)
```

Comment:
Do you mean you would prefer the two versions unoptimized and optimzed to be merged into a bigger one? We could do that, but I think it is cleaner to have 2 smaller functions that can be used by themselves, rather than a bigger one.

--- 
About `eif_file.h`
I thought the eif_...h file where the "public" part of the runtime.
And for now it is used for `eif_file_fd(f)`  and less critical `eif_file_size(f)`